optaweb-vehicle-routing-frontend/src/ui/components/SearchBox.tsx

```diff
@@ -134,6 +134,7 @@ class SearchBox extends React.Component<Props, State> {
           placeholder="Search to add a location..."
           aria-label="geosearch text input"
           onChange={this.handleTextInputChange}
+          data-cy="demo-location-text-input"
```

Comment:
I suggest a few changes:
1. Remove `demo` prefix. It doesn't have any relation to the search box component. Yes, SearchBox is incidentally only used on Demo page but once we use it somewhere else...?
2. Align it with the aria label -> `data-cy="geosearch-text-input"`.

Hmm? Can we just reuse `aria-label` for the purpose of locating elements in a Cypress test to avoid duplication and kill two elephants with one screwdriver? (Let's discuss this in person).

Reply:
ad 1) sure, good point
ad 2) it would mean that `aria-label`, which serves for accessibility, would be used for one more purpose and thus it might not be stable way of selecting elements, as the test would break each time you update the label.

But as you said, let's discuss it in person.

Reply:
1. `demo` prefix - please remove.

2. Agreed. Let's not misuse `aria-*` for locating elements in tests.